client/my-sites/checkout/composite-checkout/test/translate-cart.js

```diff
@@ -129,27 +121,6 @@ describe( 'translateResponseCartToWPCOMCart', function () {
 			} );
 		} );
 
-		describe( 'taxes', function () {
```

Comment:
It looks like we're losing these tests, which makes me a little nervous. Is it possible/sensible to move them to the new package?

Reply:
That's a good point. They were testing the moved functions implicitly, which is why I didn't notice. I'll create some tests in the package.

Reply:
I did better and added tests to cover the specific cases of the functions.